### PR TITLE
Remove hardfork build & test from nightly

### PR DESCRIPTION
**Pull request by dkijania**
Hardfork build process need to be revised and improved until next hardfork. Till we resolve all pressing issues, I'm removing hardfork packages jobs in nightly

### CODE REVIEW

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-nightly-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
Nightly: https://buildkite.com/o-1-labs-2/mina-end-to-end-nightlies/builds/2391

**Review by SanabriaRusso — APPROVED** [review]

File: buildkite/src/Jobs/Test/NixBuildTest.dhall [hunk 1]
```diff
@@ -44,6 +44,7 @@ in  Pipeline.build
                   Cmd.Docker::{
                   , image = ContainerImages.nixos
                   , privileged = True
+                  , useBash = False
```
Comment (georgeee) [inline buildkite/src/Jobs/Test/NixBuildTest.dhall:47]: Is it related?

**Pull request merged**